src/test/kotlin/no/nav/familie/ef/sak/vilkår/VurderingServiceIntegrasjonsTest.kt

```diff
@@ -120,28 +118,6 @@ internal class VurderingServiceIntegrasjonsTest : OppslagSpringRunnerTest() {
             .hasMessage("Tidligere behandling=$tidligereBehandlingId har ikke noen vilkår")
     }
 
-    @Test
```

Comment:
Har ikke alget ny integrasjonstest. Skulle vi gjort det?

Reply:
For meg så ser det ut som begge testene tester samme metode, så det er fint om bare en av de beholdes.